examples/smart-mastering-complete/flows/persons.flow.json

```diff
@@ -1,343 +1,285 @@
-            "stdAlgorithm" : {
-              "timestamp" : {
-                "path" : "/envelope/headers/createdOn"
-              }
-            },
-            "custom" : [ {
-              "name" : "mergeDob",
-              "function" : "mergeDob",
-              "at" : "/custom-modules/custom/dob-merge.sjs"
-            } ],
-            "collections" : {
-              "onNoMatch" : {
-                "add" : {
-                  "collection" : [ "no-match" ]
-                },
-                "remove" : {
-                  "collection" : [ "" ]
-                },
-                "set" : {
-                  "collection" : [ "" ]
-                }
-              },
-              "onNotification" : {
-                "add" : {
-                  "collection" : [ "likely-match" ]
-                },
-                "remove" : {
-                  "collection" : [ "" ]
-                },
-                "set" : {
-                  "collection" : [ "" ]
-                }
-              },
-              "onMerge" : {
-                "add" : {
-                  "collection" : [ "" ]
-                },
-                "remove" : {
-                  "collection" : [ "no-match", "likely-match" ]
-                },
-                "set" : {
-                  "collection" : [ "" ]
-                }
-              }
-            }
-          },
-          "mergeStrategies" : [ {
-            "sourceWeights" : [ ],
-            "name" : "retain-single-value",
-            "algorithmRef" : "standard",
-            "maxValues" : "1",
-            "length" : {
-              "weight" : "10"
-            }
-          } ],
-          "merging" : [ {
-            "propertyName" : "Address",
-            "algorithmRef" : "standard",
-            "maxValues" : "1",
-            "length" : {
-              "weight" : "10"
-            },
-            "sourceWeights" : [ ],
-            "strategy" : "retain-single-value"
-          }, {
-            "propertyName" : "DateOfBirth",
-            "algorithmRef" : "mergeDob",
-            "sourceWeights" : [ ]
-          }, {
-            "propertyName" : "ZipCode",
-            "algorithmRef" : "standard",
-            "maxValues" : "1",
-            "length" : {
-              "weight" : "10"
-            },
-            "sourceWeights" : [ ],
-            "strategy" : "retain-single-value"
-          } ]
-        },
-        "sourceCollection" : "map-persons",
-        "outputFormat" : "json",
-        "targetDatabase" : "data-hub-FINAL"
-      },
-      "customHook" : { },
-      "retryLimit" : null,
-      "batchSize" : 0,
-      "threadCount" : 0,
-      "stepDefinitionName" : "default-merging",
-      "stepDefinitionType" : "MERGING"
-    }
-  }
-}
+{
+  "name" : "persons",
+  "steps" : {
+    "1" : {
+      "stepId" : "persons-ingest-persons-ingestion"
+    },
+    "2" : {
+      "stepId" : "persons-map-persons-mapping"
+    },
+    "3" : {
+      "name" : "generate-dictionary",
```

Comment:
We'll need to migrate this project again, as we'll soon be supporting migration of custom steps. Still good to do this.

Reply:
Actually, from a tracking perspective, I think it'll be better to block this story until custom steps can be migrated. Otherwise we'll just need to redo this. I'll update the JIRA ticket to block it and note the dependency.

Reply:
Agreed. Since we should migrate once for a project, it would be better to do this once custom steps can be migrated.